translation/source/patron.xml

```diff
@@ -40,7 +40,7 @@ Then we pay a full-time developer: %s, the founder of Lichess.
   <string name="officialNonProfit">Is Lichess an official non-profit?</string>
   <string name="actOfCreation">Yes, here's the act of creation (in French)</string>
   <string name="changeMonthlySupport">Can I change/cancel my monthly support?</string>
-  <string name="changeOrContact">Yes, at any time, from this page.
+  <string name="changeOrContact">Yes, at any time, from this page whereas if you use PayPal, you can cancel through PayPal.
```

Comment:
Adding here seems good. But this sentence is unnecessarily complicated. Just adding a new sentence "If you use PayPal, ..." seems better. And maybe "you **need** to cancel through PayPal".

Also, for content changes like this, it's better to create a new translation key to force re-translation.